nginx/staging-nginx.conf

```diff
@@ -0,0 +1,35 @@
+
+user  nginx;
+worker_processes  1;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+
+    #gzip  on;
+
+    auth_basic  "PiE Website Staging";
+    auth_basic_user_file "/usr/share/nginx/auth/.htpasswd";
+
+    include /etc/nginx/conf.d/*.conf;
```

Comment:
Do we still need this, or are we relying on a pre-baked config already in the docker image to host from that html dir?

Reply:
This config is what the default config came with - I'm not too sure what the include actually picks up.

Reply:
Removed